index.js

```diff
@@ -572,3 +588,17 @@ async function* getSpaceUsageReports(client, period) {
     }
   }
 }
+
+/**
+ * @param {{ json: boolean }} options
+ */
+export async function createKey({ json }) {
+  const signer = await ed25519.generate()
+  const key = ed25519.format(signer)
+  if (json) {
+    console.log(dagJSON.stringify(signer.toArchive()))
+  } else {
+    console.log(`# ${signer.did()}`)
+    console.log(`${key}`)
```

Comment:
Is it already a string?
```suggestion
    console.log(key)
```